js/stores/appStore.js

```diff
@@ -202,6 +204,14 @@ const applyReducers = (state, action, immutableAction) => reducers.reduce(
       return newState
     }, appState)
 
+const isWebTorrentEnabled = (state) => {
```

Comment:
Place of this function is not perfect. Where would you put it, so that I can also add unit tests?

cc @bsclifton @petemill

Reply:
@NejcZdovc how about `app/common/state/extensionState.js`?

Reply:
I was thinking about that file but was not sure. Ok let me put it there. Thx

Reply:
that is a perfect place to put it- good call  😄

Reply:
moved